lib/mastodon/domains_cli.rb

```diff
@@ -27,14 +27,14 @@ def self.exit_on_failure?
       from a single domain, all accounts from domains that are not whitelisted
       are removed from the database.
     LONG_DESC
-    def purge(domain = nil)
+    def purge(*domains)
       dry_run = options[:dry_run] ? ' (DRY RUN)' : ''
 
       scope = begin
         if options[:whitelist_mode]
           Account.remote.where.not(domain: DomainAllow.pluck(:domain))
-        elsif domain.present?
-          Account.remote.where(domain: domain)
+        elsif domains.present?
+          Account.remote.where('domain in (?)', domains)
```

Comment:
You don't have to write SQL, if you use `where(domain: domains)` it will do an `IN` query.